lib/xebow/keys.ex

```diff
@@ -156,12 +157,14 @@ defmodule Xebow.Keys do
 
   # Custom Key Functions
 
-  def flash_red do
-    Xebow.RGBMatrix.flash("red")
-  end
+  def flash(color) do
```

Comment:
(no action)

Nice refactor! I like how you pulled `flash` out of the `RGBMatrix` and moved it into the "application". At some point we should probably separate the application layer (the key presses and demo logic) from the library/hardware. For example, the demo's animations should probably be separated from the key handling code. But I think that can be a problem for another PR.